web/embed-flutter.html

```diff
@@ -30,6 +30,13 @@
 
     <script src="packages/codemirror/codemirror.js" defer></script>
     <script src="packages/codemirror/addon/scroll/simplescrollbars.js" defer></script>
+    <script src="packages/codemirror/addon/fold/foldcode.js" defer></script>
+    <script src="packages/codemirror/addon/fold/foldgutter.js" defer></script>
+    <script src="packages/codemirror/addon/fold/brace-fold.js" defer></script>
+    <script src="packages/codemirror/addon/fold/xml-fold.js" defer></script>
+    <script src="packages/codemirror/addon/fold/indent-fold.js" defer></script>
+    <script src="packages/codemirror/addon/fold/comment-fold.js" defer></script>
+    <script src="packages/codemirror/addon/edit/matchtags.js" defer></script>
```

Comment:
Are these also included in codemirror after https://github.com/dart-lang/dart-pad/pull/2214?

Reply:
Yes they are included in the new codemirror.  So those script includes are no longer required and should now be removed.

Reply:
(from all of the html files)

Reply:
i resolved the conflicts taking the new codemirror ane now merged include file changes into account.